### DISCUSSION TITLE
Awesome job

### DISCUSSION THREAD

**Discussion by sylvaingirardbe** [community/community#9240] [Copilot Conversations]
Just for a change in this list with questions and complaints: you guys did an awesome job with Copilot, already. 

It suggests fairly big pieces of code (often times I didn't even know I would need it yet!) very well but what really helps out a lot are the smaller code completions like this one:
![image](https://user-images.githubusercontent.com/1567350/147205594-be00dad8-96ea-45f2-addf-4bfbd9f518cd.png)

There's no tooling that provides this kind of intellisense except for Copilot and it speeds up things quite a bit.

Keep up the awesome work and thanks!

**Comment by prezire** [comment]
I second that. I'm always expecting syntax completions but Copilot offers me the business logic I needed instead. AWESOME work guys! Keep it up.

**Reply by sophos-rickc** [reply]
I'll jump on this bandwagon, too. Copilot is the code assistant I always wanted. As I get older -- I've been at this 40+ years -- Copilot saves a lot of typing for these stiff fingers, LOL! 

I do have one request. Tab-completion doesn't seem to work when editing Markdown files... ;)

**Reply by Urethramancer** [reply]
It pretty much wrote all my SQL setup, and about half the functions for handling the DB in a current project. I sometimes have to delete a suggestion and write smarter queries, but it saves me hours of looking up how to use statements, DB APIs and even fills out a lot of the logic required for what it retrieves or creates.
